metro.config.js

```diff
@@ -38,6 +42,36 @@ const config = {
       },
     }),
   },
+  server: {
+    enhanceMiddleware: (middleware) => {
+      // This only runs once when Metro starts
+      if (!spawned) {
+        spawned = true
+        childProcess = spawn(
+          "node",
+          ["-e", "require('./standalone-server').startReactotronServer({ port: 9292 })"],
+          {
+            stdio: "inherit", // pipe output directly to console
+            shell: false,
+          },
+        )
+
+        childProcess.on("exit", (code) => {
+          console.log(`Child process exited with code ${code}`)
+        })
+
+        // Handle SIGINT (Ctrl+C) to terminate child process
+        process.on("SIGINT", () => {
+          if (childProcess) {
+            console.log("Terminating child process...")
+            childProcess.kill("SIGINT")
+          }
+        })
+      }
+
+      return middleware
+    },
+  },
```

Comment:
We can just run it in the metro process, and it'll make this a lot simpler.

```suggestion
  require('./standalone-server').startReactotronServer({ port: 9292 })
```